core/dbt/parser/partial.py

```diff
@@ -212,6 +212,10 @@ def update_in_saved(self, file_id):
     def update_mssat_in_saved(self, new_source_file, old_source_file):
 
         # These files only have one node.
```

Comment:
Should this comment be below the `if` statement with the original code it was with?

Reply:
Yes. Addressed in latest version.